config/settings.yml

```diff
@@ -12,6 +12,13 @@ financial:
     csv: true
     xml: true
 
+converters:
+  default:
+    product_to_journal_rows: Converters::ProductToJournalRows
+    order_detail_to_journal_rows: Converters::OrderDetailToJournalRows
```

Comment:
I would agree that's somewhat redundant in that case, but I still feel `Converters` is overly general. I'd maybe go with `JournalRowConverters::FromOrderDetail` and `JournalRowConverters::FromProduct`. Does that make sense?